.github/ISSUE_TEMPLATE/bug_report.md

```diff
@@ -5,6 +5,23 @@ labels: bug
 
 ---
 
+<!--
+  ___ __  __ ____   ___  ____ _____  _    _   _ _____ 
+ |_ _|  \/  |  _ \ / _ \|  _ \_   _|/ \  | \ | |_   _|
+  | || |\/| | |_) | | | | |_) || | / _ \ |  \| | | |  
+  | || |  | |  __/| |_| |  _ < | |/ ___ \| |\  | | |  
+ |___|_|  |_|_|    \___/|_| \_\|_/_/   \_\_| \_| |_|     
+
+IF YOU ARE REPORTING A CHEAT OR A SECURITY VULNERABILITY... STOP!
+
+DO NOT MAKE A PUBLIC POST. GO BACK. CLICK THE 'VIEW POLICY' BUTTON.
+
+IF YOU ARE LOST AND CANNOT FIND A LINK, HERE IT IS:
```

Comment:
Looks good until here,
```suggestion
    IF YOU HAVE TROUBLE FINDING THE RIGHT LINK:
```

Reply:
because the other sounds a little too harsh

Reply:
thanks, good call